### PR TITLE
Update dioxus dependency to 0.5

### PR DESCRIPTION
**Pull request by Stygmates**
Updates the crate to support dioxus 0.5.
The example runs locally with the updated dependency so it should be good.

### CODE REVIEW

File: Cargo.toml
Comment (marc2332) [inline Cargo.toml:1]:
```toml
[dependencies]
dioxus = { version = "0.5" }
pulldown-cmark = "0.9.3"

[dev-dependencies]
dioxus = { version = "0.5", features = ["desktop"] }
```

File: src/lib.rs [hunk 1]
```diff
@@ -3,28 +3,29 @@
...
+pub struct MarkdownProps {
     #[props(default)]
-    id: &'a str,
+    id: Signal<String>,
     #[props(default)]
-    class: &'a str,
+    class: Signal<String>,
 
-    content: &'a str,
+    content: Signal<String>,
```
Comment (marc2332) [inline src/lib.rs]: I would use ReadOnlySignal instead.

File: examples/bulma.rs [hunk 2]
```diff
@@ -2,21 +2,17 @@ use dioxus::prelude::*;
...
 fn main() {
-    dioxus_desktop::launch(app);
+    dioxus::launch(app);
 }
 
-fn app(cx: Scope) -> Element {
-    cx.render(rsx! {
+fn app() -> Element {
+    let class = use_signal(|| String::from("content"));
+    let content = use_signal(|| String::from(include_str!("../README.md")));
```
Comment (marc2332) [inline examples/bulma.rs]: No need to wrap `include_str!("../README.md")` in a signal if you change the ReadOnlySignal thing

File: examples/bulma.rs [hunk 3]
```diff
@@ -2,21 +2,17 @@ use dioxus::prelude::*;
 use dioxus_markdown::Markdown;
 
 fn main() {
-    dioxus_desktop::launch(app);
+    dioxus::launch(app);
```
Comment (marc2332) [inline examples/bulma.rs]:
```suggestion
    launch(app);
```

**Review by marc2332 — CHANGES_REQUESTED** [review]

**Comment by Stygmates** [comment]
Thank you for your review! I think I applied all of your suggestions, let me know if I need to change anything else.

**Comment by jkelleyrtp** [comment]
Looks good, thanks!

**Comment by ifsheldon** [comment]
Hey! I think you brought a silent breaking change here.  @Stygmates 
https://github.com/DioxusLabs/markdown/blame/5ca15aa634de28b9241c2e0acc863231f5618107/Cargo.toml#L9
My web code no longer complies due to the `desktop` feature here. It took me a while to find out the mystery `tokio` dependency that showed up

**Comment by marc2332** [comment]
> Hey! I think you brought a silent breaking change here. @Stygmates https://github.com/DioxusLabs/markdown/blame/5ca15aa634de28b9241c2e0acc863231f5618107/Cargo.toml#L9 My web code no longer complies due to the `desktop` feature here. It took me a while to find out the mystery `tokio` dependency that showed up

Seems like my [comment](https://github.com/DioxusLabs/markdown/pull/6#discussion_r1551320617) was not applied

**Comment by marc2332** [comment]
> Hey! I think you brought a silent breaking change here. @Stygmates https://github.com/DioxusLabs/markdown/blame/5ca15aa634de28b9241c2e0acc863231f5618107/Cargo.toml#L9 My web code no longer complies due to the `desktop` feature here. It took me a while to find out the mystery `tokio` dependency that showed up

Opened https://github.com/DioxusLabs/markdown/pull/7

**Pull request merged**